DSNP-Wallet/Source/Modules/GenerateKeysViewController.swift

```diff
@@ -18,11 +18,13 @@ class GenerateKeysViewController: UIViewController {
     }
     
     private func manageKeys() {
-        if let keys = AuthManager.shared.loadKeys() {
+        
+        do {
+            let keys = try DSNPWallet().loadKeys()
```

Comment:
@ryansheh Should we be using `loadKeys()` here or the `fetchKey()` on the `AccountManager`? I notice this func wants DSNP keys but the other function returns `Data`

Reply:
This class is actually safe to remove. It was used as a placeholder vc for testing until this  onboarding epic was made.